types/dropzone/index.d.ts

```diff
@@ -46,6 +46,12 @@ declare namespace Dropzone {
         upload?: DropzoneFileUpload;
     }
 
+    export interface DropzoneMockFile {
+        name: string;
+        size: number;
```

Comment:
Source code says it also [needs `imageURL`](https://gitlab.com/meno/dropzone/-/blob/v5.7.0/src/dropzone.js#L1679-1681), but that's not correct. It's never read and `imageURL` is passed into `displayExistingFile` directly.

Reply:
Ya that might be a outdated code comment